beachball.config.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ * @format
+ * @ts-check
+ */
+
+const {execSync} = require('child_process');
+
+module.exports = {
+  ...require('@rnw-scripts/generated-beachball-config'),
+  
+  // Do not geneerate tags for monorepo packages by default, to avoid a GitHub
+  // release for every package.
+  gitTags: false,
+
+  // Default commit message, used if we do not detect an RNW version update
+  message: 'Publish Packages | ***NO_CI***',
+
+  hooks: {
+    // Stamp versions when we publish a new package
+    prepublish: (_packagePath, name, version) => {
+      if (name === 'react-native-windows') {
+        execSync(`yarn stamp-version ${version}`);
+        execSync('git add --all');
+        execSync(`git commit --amend --message "Publish Packages (${version}) | ***NO_CI***"`);
```

Comment:
Need to double check this is sequenced the way I want it to be in regards to temporary branch state